source/postprocess/mobility_statistics.cc

```diff
@@ -66,8 +66,9 @@ namespace aspect
       std::vector<Tensor<1,dim>> surface_velocity_values (n_q_points_face);
 
       double local_velocity_square_integral = 0;
-      double local_velocity_square_integral_top_boundary;
-      double local_top_boundary_area;
+      double local_velocity_square_integral_top_boundary = 0;
+      double local_top_boundary_area = 0;
```

Comment:
Ah, great catch. Well done.

Out of curiosity, how did you find this?

Reply:
I saw that it was the test `simple_shear_output_the_mobility` that would occasionally fail in PRs, but pass when rerun (or run locally). The test was added recently as part of #4389 so I knew to look into that postprocessor, and that it was something that is essentially non-deterministic. I am actually surprised we did not get a compiler warning about uninitialized use.